project/src/main/webapp/user-home.html

```diff
@@ -8,6 +8,7 @@
     <link rel="stylesheet" href="main.css" />
     <script type="module" src="./components/user-home/nav-panel.js"></script>
     <script type="module" src="./components/user-home/docs-component.js"></script>
+    <script src="https://www.gstatic.com/firebasejs/5.5.4/firebase.js"></script>
```

Comment:
whats the gstatic?

Reply:
nav-panel.js needs access to firebase to generate a new hash. This imports firebase.